rollbar.js

```diff
@@ -2,11 +2,55 @@
 
 "use strict";
 
-var api = require('./lib/api');
-var notifier = require('./lib/notifier');
-var parser = require('./lib/parser');
+var Api = require('./lib/api');
+var Notifier = require('./lib/notifier');
 
-var initialized = false;
+var Rollbar = function() {
+  var api, notifier;
+  var initialized = false;
+
+  Object.defineProperties(this, {
+    _api: {
+      set: function(val) {
+        api = val;
+      }
+    },
+    _initialized: {
+      set: function(val) {
+        initialized = val;
+      }
+    },
+    _notifier: {
+      set: function(val) {
+        notifier = val;
+      }
+    },
+    api: {
+      enumerable: true,
+      get: function() {
+        return api;
+      }
+    },
+    initialized: {
+      enumerable: true,
+      get: function() {
+        return initialized;
+      }
+    },
+    notifier: {
+      enumerable: true,
+      get: function() {
+        return notifier;
+      }
+    },
+  });
+}
+
+Rollbar.prototype.create = function(accessToken, options) {
```

Comment:
I'm more inclined to say that you should put this functionality into the constructor since the instance returned from `create()` will have no shared state with the instance you call `create()` on. 

Eventually when we get around to implementing the new interface methods, (like rollbar.js and rollbar-gem already have) this will be achieved with:

```js
var Rollbar = require("rollbar");
Rollbar.configure({accessToken: "ACCESS_TOKEN_1"});
var Rollbar2 = Rollbar.scope();
Rollbar2.configure({accessToken: "ACCESS_TOKEN_2"});
```

Until then, I think it makes sense for the rollbar.js module to return an object which initially just has an `init()` method. That method will create and initialize a new Notifier instance and create proxy methods which will call the associated methods on the new Notifier instance. That way, people can continue using rollbar.js as a singleton.

For creating new instances with different access tokens, the object exported by rollbar.js will also have a constructor that will return a new instance.

```js
// For multiple instances
var rollbar = require("rollbar");
var Rollbar1 = new rollbar("ACCESS_TOKEN_1");
var Rollbar2 = new rollbar("ACCESS_TOKEN_2");
```

or

```js
// Using a singleton
var Rollbar = require("rollbar");
Rollbar.init("ACCESS_TOKEN_1");
```

This will mean that you cannot create the `globalNotifier` in rollbar.js.